src/Misc/layoutbin/powershell/Add-AzureGuestAgentCapabilities.ps1

```diff
@@ -0,0 +1,24 @@
+[CmdletBinding()]
+param()
+
+$guestAgentPath = $null
+
+try {
+    $proc = Get-Process 'WindowsAzureGuestAgent'
```

Comment:
Ibiza flow might use Network service account during agent configuration. Can you check if you get the desired result with it?